zebra-consensus/src/block/subsidy/general.rs

```diff
@@ -25,37 +25,35 @@ pub fn halving_divisor(height: Height, network: Network) -> Option<u64> {
         .activation_height(network)
         .expect("blossom activation height should be available");
 
-    if height < SLOW_START_SHIFT {
+    let slow_start_shift_diff = height - SLOW_START_SHIFT;
+    let slow_start_shift_diff = slow_start_shift_diff.unwrap_or_else(|| {
```

Comment:
Optional: use `let .. else` statement here, i.e.

```suggestion
    let Some(slow_start_shift_diff) = height - SLOW_START_SHIFT else {
```